File: Practica-CodigosJava/Ej5.java
//Dados dos numeros enteros positivos, determinar si uno es multiplo del otro
import java.util.Scanner;
public class Ej5 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Ingrese numero 1: ");
        int n1 = scanner.nextInt();
        System.out.print("Ingrese numero 2: ");
        int n2 = scanner.nextInt();
        if (n1 % n2 == 0) {
            System.out.println(n1 + " es multiplo de " + n2);
        } else {
            System.out.println(n1 + " no es multiplo de " + n2);
        }
        if (n2 % n1 == 0) {
            System.out.println(n2 + " es multiplo de " + n1);
        } else {
            System.out.println(n2 + " no es multiplo de " + n1);
        }
        scanner.close();
    }
}

File: Practica/Ej5.py
#Dados dos nuemros enteros positivos, determinar si uno es multiplo del otro
n1=int(input("Ingrese numero 1: "))
n2=int(input("Ingrese numero 2: "))
if n1 % n2 == 0:
    print(n2,"es multiplo de",n1)
else:
    print(n2,"no es multiplo de",n1)
if n2 % n1 == 0:
    print(n1,"es multiplo de",n2)
else:
    print(n1,"no es multiplo de",n2)
